src/astro/sql/operators/dataframe.py

```diff
@@ -81,7 +79,7 @@ def __init__(
         :param warehouse: (Snowflake) Which warehouse to use for the input table
         :param kwargs:
         """
```

Comment:
We should also add what this operator returns (df or table) so users know what to expect. And this will be helpful when the Sphinx docs are published